src/vsarVSTS-Roll-up-board/overview.md

```diff
@@ -41,4 +41,4 @@ We need your feedback! Here are some ways to connect with us:
 - Add a comment in the Q&A section below and please add an actionable review.
 - Send us an [email](mailto://mktdevlabs@microsoft.com).
 
-Review the [list of features and resolved issues of latest tools and extensions](https://aka.ms/vsarreleases) for information on the latest releases.
\ No newline at end of file
+Review the [list of features and resolved issues of latest tools and extensions](https://aka.ms/vsarreleases) for information on the latest releases.
```

Comment:
We will need to migrate this page, or re-create a similar one on the alm DevOps rangers site.

Reply:
I am removing the reference for now.